package.json

```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "@astrojs/react": "1.1.0",
     "@astrojs/tailwind": "1.0.0",
+    "@iconify/json": "^2.1.100",
```

Comment:
Wersje paczek powinny być przypięte (bez `^`). Możliwe, że coś źle skonfigurowałem skoro nie zainstalowało się tak z automatu

Reply:
Zrobiłeś dobrze, ja to z łapki wpisałem. Oni tą paczuszkę co jakiś czas aktualizują, w ten sposób nasi użytkownicy zawsze ( czyli do wersji 3.0) dostaną świeżutkie ikonki bez żadnego utrzymywania po naszej stronie

Reply:
Dobry pomysł, niemniej na dniach skonfiguruję nam automatyczne aktualizacje wszystkich paczek co tydzień, także `^` może nie zrobić dużej różnicy

Reply:
oki, cofnąłem